create_employee.sql

```diff
@@ -0,0 +1,62 @@
+-- 1. As a member of the HR team I want to be able to create a new sales employee. 
+-- I should be able to store a name, salary, bank account number, national insurance number and commission rate
+CREATE TABLE employee (
+    employee_id int PRIMARY KEY AUTO_INCREMENT,
+    f_name varchar(255),
+    l_name varchar(255),
+    salary decimal,
```

Comment:
default of decimal is (10,0), which means you wouldn't be able to store anything after decimal place, are you happy with this?